### DISCUSSION TITLE
[Q/A][Java SDK (Component)] Does version 3.2.2 support ziplin access

### DISCUSSION THREAD

**Discussion by 3Taste** [apache/dubbo#14428] [Question]
### Pre-check

- [X] I am sure that all the content I provide is in English.

### Apache Dubbo Component

Java SDK (apache/dubbo)

### Details

I configured ziplin support according to the documentation on the website, but it didn't work. I see the official example with version 3.3.0:
https://cn.dubbo.apache.org/zh-cn/overview/tasks/observability/tracing/zipkin/

My local dubbo version: 3.2.2
jdk version: 17
I would like to know if there is no functional support for version 3.2.2

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by AlbumenJ** [comment]
@conghuhu PTAL
